docs/volume_annotation.md

```diff
@@ -36,6 +36,8 @@ The "merger mode" is available in skeleton and hybrid tracing mode. Mark connect
 ### Mappings / On-Demand Agglomeration
 With webKnossos it is possible to apply a precomputed agglomeration file to combine over-segmented volume annotations on demand. This alleviates the demand to materialize agglomeration results prematurely and allows researchers to apply different agglomerations to the data for experimentation. We typically produce several agglomeration results based on different thresholds and methods and use webKnossos to quickly review these results on subsets of a dataset in an interactive session.
 
+{% embed url="https://www.youtube.com/watch?v=ZmUqyIoA9Gw "%}
```

Comment:
You forgot to remove the line embedding this video above the mappings headline, once that is removed this should be good to merge :)

Reply:
I removed it :)